clash-prelude/src/Clash/Sized/Fixed.hs

```diff
@@ -69,6 +69,7 @@ operator that uses truncation introduces an additional error of /0.109375/:
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE RoleAnnotations #-}
```

Comment:
Don't forget to remove this if you decide to remove the role annotation for `Fixed`